learn/wavemaker-release-notes/v10-7-0.md

```diff
@@ -42,20 +44,21 @@ sidebar_label: "Release v10.7.0"
 
 ### Query Optimization for Faster DB Imports
 
-- Metadata queries invoked to fetch tables' constraints information are optimized for ~600x faster database imports in an application.
+- Metadata queries invoked to fetch tables' constraints information are optimized with ~600x faster database imports in an application.
```

Comment:
Importing Databases into WaveMaker application is now much faster. We have optimised the queries that were used to read the metadata of the databases.

Reply:
can you rephrase this sentence like i suggested above. We should explain what the impact of the improvement is @swethakundaram